frontend/.pa11yci.json

```diff
@@ -0,0 +1,100 @@
+{
+  "defaults": {
+    "timeout": 240000,
+    "runners": ["axe"],
+    "ignore": ["color-contrast"],
+    "concurrency": 1,
+    "chromeLaunchConfig": {
+      "ignoreHTTPSErrors": true,
+      "args": ["--disable-dev-shm-usage", "--no-sandbox"]
+    }
+  },
+  "urls": [
+    {
+      "url": "http://localhost:3000",
+      "viewport": { "width": 320, "height": 480 },
+      "actions": [
+        "wait for element #main-content to be visible",
+        "screen capture screenshots-output/mobile-main-view.png",
+        "click element .usa-navbar button",
+        "screen capture screenshots-output/mobile-expand-menu.png"
+      ]
+    },
+    {
+      "url": "http://localhost:3000",
+      "actions": [
+        "wait for element #main-content to be visible",
+        "screen capture screenshots-output/desktop-main-view-home.png"
+      ]
+    },
+    {
+      "url": "http://localhost:3000/search?status=forecasted,posted",
+      "viewport": { "width": 320, "height": 480 },
```

Comment:
If we are going to set viewport to width 320 and height 480 on all components we can just set that as a default and then change on the few components where we don't want that behavior